wsdl/types.go

```diff
@@ -62,12 +66,54 @@ type Union struct {
 	MemberTypes string   `xml:"memberTypes,attr"`
 }
 
-// Restriction describes the WSDL type of the simple type and
+// Attribute is a "meta" WSDL element it belongs to a Restriction and marks a an arbitrary value with its key "ref"
+// Only a monster chooses to give an XML Element the name "Attribute", but there you have it
+type Attribute struct {
```

Comment:
If this is only for the Restriction type below, can we name it RestrictionAttr instead?